src/main/java/hudson/tasks/junit/SuiteResult.java

```diff
@@ -348,4 +349,19 @@ void setParent(hudson.tasks.junit.TestResult parent) {
     private static final long serialVersionUID = 1L;
 
     private static final Pattern SUREFIRE_FILENAME = Pattern.compile("TEST-(.+)\\.xml");
+
+    /**
+     * Merges another SuiteResult into this one.
+     * 
+     * @param sr
+     */
+    public void merge(SuiteResult sr) {
+        if (hasTimeAttr) {
```

Comment:
...i think we will have time attributest on all testuites or neither of them...i didn't wanted to get lost in the mixed case...but if you think it's important i can fix that ;)